tools/tree-viz.html

```diff
@@ -8,7 +8,7 @@
     <script type="importmap">
       {
         "imports": {
-          "@grafana/lezer-logql": "https://cdn.jsdelivr.net/npm/@grafana/lezer-logql@0.0.17/index.es.js",
+          "@grafana/lezer-logql": "https://cdn.jsdelivr.net/npm/@grafana/lezer-logql@0.0.16/index.es.js",
```

Comment:
Shouldn't this be changed to 18, so it has the most up to date package?

Reply:
Good catch! Changed to `0.0.18`.